tests/integration/modules/test_win_pkg.py

```diff
@@ -33,8 +34,24 @@ def test_adding_removing_pkg_sls(self):
         Test add and removing a new pkg sls
         in the windows software repository
         '''
-        def _check_pkg(pkgs, exists=True):
-            self.run_function('pkg.refresh_db')
+        def _check_pkg(pkgs, exists=True, check_refresh=None):
```

Comment:
Should `check_refresh` be a boolean by default?

Reply:
I could change it to boolean but we are passing in `check_refresh=2`, when using this so i kept it at None. Is it better practice to keep it a boolean even though we are not passing True/False into the kwarg.